.env

```diff
@@ -1,16 +0,0 @@
-# development credentials assume application is running on port 7777
```

Comment:
You don't need to remove this file. It is intended for development and only contains development specific credentials.